### PR TITLE
Update to Haiku Section

### PR DESCRIPTION
**Pull request by AleksandraZhi**
**Poetry/Haiku Section Update:**

Instead of having all the haikus on one page, to make it more visually friendly - substituted it with a carousel, that contains 4 haiku in each slide.

haiku section

### CODE REVIEW

File: server/views/includes/index-poetry.pug [hunk 1]
```diff
@@ -0,0 +1,139 @@
+.row 
+    .col
+        div(id="carouselExampleControls" class="carousel slide" data-ride="carousel")
```
Comment (mforner13) [inline server/views/includes/index-poetry.pug]: Should example still be in this id name? :)

File: server/views/includes/index-poetry.pug [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                        .col
+                            p 
+                                | Принятие…
+                                br
+                                | Подожди, я с тобой попрощаюсь.
+                                br
+                                | Эхо в пустоте
+                        .col-2
+
+            a(class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev")
```
Comment (mforner13) [inline server/views/includes/index-poetry.pug]: I don't know whether it's just that I can't find the element with this ID - but where would this take you to? :)

File: server/views/index.pug [hunk 3]
```diff
@@ -6,316 +6,217 @@ html
     title="Ch-Cat.ru"
 
     link(rel='stylesheet', href='/css/main-styles.css')
-    link(rel='stylesheet', href='/bootstrap/dist/css/bootstrap.min.css')
+    link(rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous")
```
Comment (mforner13) [inline server/views/index.pug:9]: Good idea to use the cdn instead! Then hopefully no maintenance if something in the library changes 👍 😄

**Review by mforner13 — COMMENTED** [review]
Looks great 😄 To make the PR a bit cleaner you could:

- Fixup the commit messages to use the imperative
- Add a bit of formatting to the PR description to make it a bit easier to read :) E.g. using spacing/bold/headings

A good pattern to follow for how to write PR messages in general is:

- Why I'm making this change
- What the change does, both in terms of user outcomes and in terms of the code
- Why I've taken this approach, showing why you rejected other approaches to coding the story

File: server/views/includes/index-poetry.pug (hunk 1 above)
Comment (AleksandraZhi) [reply server/views/includes/index-poetry.pug]: Updated the ID name to not include "example"

File: server/views/includes/index-poetry.pug (hunk 2 above)
Comment (AleksandraZhi) [reply server/views/includes/index-poetry.pug]: href in line nr 134 takes you to the previous slide of "#carouselExampleIndicators" (now changed to "carouselIndicators") - it returns you to the div in the top of the carousel element - line nr 3. It's a Bootstrap component with Javascript behind the scenes.

**Comment by AleksandraZhi** [comment]
> Looks great 😄 To make the PR a bit cleaner you could:
> 
> * Fixup the commit messages to use the imperative
> * Add a bit of formatting to the PR description to make it a bit easier to read :) E.g. using spacing/bold/headings
> 
> A good pattern to follow for how to write PR messages in general is:
> 
> * Why I'm making this change
> * What the change does, both in terms of user outcomes and in terms of the code
> * Why I've taken this approach, showing why you rejected other approaches to coding the story

Thank you, I've updated the commit messages.
Is it okay for me to merge? I left one conversation as unresolved - maybe you want to add smth after my update/comment.

**Comment by mforner13** [comment]
Thanks for making those changes! :) For the commit messages - for them to be in the imperative they'd need to be `Update ...` as currently they're in the past tense.

**Comment by AleksandraZhi** [comment]
> Thanks for making those changes! :) For the commit messages - for them to be in the imperative they'd need to be `Update ...` as currently they're in the past tense.

Noted, sorry, I misunderstood the first time.

**Pull request merged**